src/reports/components/report-sections/report-footer-text.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { EnvironmentInfo } from 'common/environment-info-provider';
+import { NamedFC } from 'common/react/named-fc';
+import { toolName } from 'content/strings/application';
+import { ToolLink } from 'reports/components/report-sections/tool-link';
+
+export type ReportFooterTextProps = EnvironmentInfo;
+
+export type ReportFooterText = React.FC<EnvironmentInfo>;
+
+export const DefaultFooterText: ReportFooterText = NamedFC<ReportFooterTextProps>(
+    'DefaultFooterText',
+    ({ axeCoreVersion, browserSpec, extensionVersion }) => (
+        <>
+            This automated checks result was generated using {`${toolName} ${extensionVersion} (Axe ${axeCoreVersion})`}, a tool that helps
```

Comment:
for service report this will say, that the report was generated using "Accessibility Insights for Web", since toolName here is not configurable